core/src/main/java/org/jdbi/v3/core/statement/BindListEmptyHandling.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.jdbi.v3.core.statement;
+
+import java.util.function.BiConsumer;
+
+/**
+ * describes what should be done if the value passed to {@link SqlStatement#bindList} is null or empty
+ */
+public enum BindListEmptyHandling implements BiConsumer<SqlStatement, String> {
+    /**
+     * <p>Output "" (without quotes, i.e. nothing).</p>
+     * <p>
+     * {@code select * from things where x in ()}
+     */
+    VOID {
+        @Override
+        public void accept(SqlStatement stmt, String key) {
+            stmt.define(key, "");
+        }
+    },
+    /**
+     * <p>Output "null" (without quotes, as keyword), useful e.g. in postgresql where "in ()" is invalid syntax.</p>
```

Comment:
The usual way to indicate a keyword is `{@code null}`

Reply:
The problem here is the perspective to document from: do we document from the perspective of what we are binding, or from what we will output in the query? Both are familiar to users and neither seems like the less important one to me, though if I absolutely had to choose I guess I would choose the query POV.

Reply:
I just think it's strange to put a quotation mark and then immediately undo it in a parenthetical, when there's an out-of-the-box style for `keyword`.  I don't feel too strongly about it either way.

Reply:
Yeah but that's not the issue... We _define_ a _string_ `"null"`, which _renders_ to the _keyword_ `null`. It does both. The question is which POV do we document from? If I had to guess I'd document the query content since that's what people will mostly need to think about.